openslides/core/static/js/core/site.js

```diff
@@ -764,18 +766,26 @@ angular.module('OpenSlidesApp.core.site', [
             if (!$scope.active_projector) {
                 $scope.changeProjector($scope.projectors[0]);
             }
-
+            $scope.getDefaultProjector();
             // stop ALL interval timer
             cancelIntervalTimers();
 
             rebuildAllElements();
         });
+        $scope.getDefaultProjector = function () {
+            var projectiondefault = ProjectionDefault.filter({name: 'agenda_current_list_of_speakers'})[0];
```

Comment:
This line is not easy to understand. What do you mean by defaultProjector in this general context? Why do we use the first element of the filter array. You may use ProjectionDefault.get() instead. At least add a comment what this function is doing.